src/scenarios/shared/mauisharedpython.py

```diff
@@ -14,12 +15,24 @@ def remove_aab_files(output_dir="."):
         if file.endswith(".aab"):
             os.remove(os.path.join(output_dir, file))  
 
-def generate_maui_rollback_dict():
-    # Generate and use rollback based on Version.Details.xml
-    # Generate the list of versions starts to get and the names to save them as in the rollback.
-    # These mapping values were taken from the previously generated rollback files for the maui workload. There should be at least one entry for each
-    # of the Maui Workload dependencies in the /eng/Version.Details.xml file, aside from Microsoft.NET.Sdk.
-    # If there are errors in the future, reach out to the maui team.
+def generate_maui_rollback_dict(target_framework: str):
```

Comment:
I thought that this flow (from `install_versioned_maui`) is currently not active. Do we plan to switch back to this path?